.github/CODEOWNERS

```diff
@@ -1,5 +1,30 @@
-* @DataDog/apm-ruby
+* @DataDog/ruby-guild
 
 # Public Documentation
 /docs/Compatibility.md @DataDog/documentation
-/docs/GettingStarted.md @DataDog/documentation
\ No newline at end of file
+/docs/GettingStarted.md @DataDog/documentation
+
+# Library
+lib/datadog/appsec/* @DataDog/asm-ruby
+lib/datadog/appsec.rb @DataDog/asm-ruby
+lib/datadog/tracing/* @DataDog/tracing-ruby
+lib/datadog/tracing.rb @DataDog/tracing-ruby
+lib/datadog/opentelemetry/* @DataDog/tracing-ruby
+lib/datadog/opentelemetry.rb @DataDog/tracing-ruby
+lib/datadog/opentracer/* @DataDog/tracing-ruby
+lib/datadog/opentracer.rb @DataDog/tracing-ruby
+lib-injection/* @DataDog/tracing-ruby
```

Comment:
I am questioning this a bit: while initially developed by tracing automatic injection sounds like a feature that applies to everyone, not just tracing.